### DISCUSSION TITLE
Compare Presets

### DISCUSSION THREAD

**Discussion by CodyPrince** [OrcaSlicer/OrcaSlicer#1090] [Ideas]
In superslicer and prusaslicer one can compare presets by going to window/compare presets. This works for print settings, filament settings, and printer settings. Hope to see it migrated to orcaslicer someday.

**Comment by hliebscher** [comment]
it is the same at Orca ...
![compare](https://github.com/SoftFever/OrcaSlicer/assets/9274162/35eb32b0-9309-4956-b116-c710bb0f1fa3)

**Reply by CodyPrince** [reply]
oh heck yeah. I was missing this the whole time. Now I love ocraslicer even more. 

Thanks, Cody

**Comment by nagyrobi** [comment]
See https://github.com/SoftFever/OrcaSlicer/issues/6080#issuecomment-2223626897
